### PR TITLE
Signals: Handle `KeyError` for no date for live_data 

### PR DESCRIPTION
**Pull request by parmarsuraj99**
If there is no date in `full_data` for `date_string`, copy the structure of `full_data` for later concatenation with `thursday_data`.

### CODE REVIEW

File: signals/example_model.py [hunk 1]
```diff
@@ -151,8 +151,12 @@ def main():
...
     last_friday = datetime.now() + relativedelta(weekday=FR(-1))
     date_string = last_friday.strftime('%Y-%m-%d')
-
-    live_data = full_data.loc[date_string].copy()
+    
+    try:
+        live_data = full_data.loc[date_string].copy()
+    except KeyError as e:
+        print(f"No ticker on {e}")
+        live_data = pd.DataFrame(columns=full_data.reset_index().columns).set_index('date')
```
Comment (jeethu) [inline signals/example_model.py:159]: This could be simpler.
```suggestion
        live_data = full_data.iloc[:0].copy()
```

**Review by jeethu — CHANGES_REQUESTED** [review]
Thanks for the PR @parmarsuraj99 ! I've got one minor nit.

File: signals/example_model.py (hunk 1 above)
Comment (parmarsuraj99) [reply signals/example_model.py]: Thanks @jeethu . Committed.

**Comment by jeethu** [comment]
Great! @parmarsuraj99 Now, could I ask you to squash those 3 commits into a single commit? 🙂

**Review by jeethu — APPROVED** [review]
LGTM!

**Comment by michael-phillips-data** [comment]
Thanks you so much both of you :)

**Pull request merged**